tests/rules/pds3/test_covims_8xxx.py

```
@@ -0,0 +1,83 @@
+            for version_pdsf in pdsf.all_versions().values():
+                assert version_pdsf.abspath in opus_id_abspaths
+
+            # Every viewset is in the product set
+            for viewset in pdsf.all_viewsets.values():
+                for viewable in viewset.viewables:
+                    assert viewable.abspath in opus_id_abspaths
+
+            # Every associated product is in the product set except metadata
+            for category in ('volumes', 'previews', 'diagrams'):
+                for abspath in pdsf.associated_abspaths(category):
+                    if '.' not in os.path.basename(abspath): continue   # skip dirs
```

Ruff (0.15.21)

[error] Multiple statements on one line (colon)

(E701)